src/components/SupportRequestModal.js

```diff
@@ -14,91 +14,98 @@ import validate from 'validate.js'
 
 
 const constraints = {
-  email: { email: true },
+  name: { presence: { allowEmpty: false } },
   subject: { presence: { allowEmpty: false } },
-  description: { presence: { allowEmpty: false } }
+  description: { presence: { allowEmpty: false } },
+  email: { email: true, presence: { allowEmpty: false } }
 }
 
-const SupportRequestModal = _.flow(
-  ajaxCaller,
-  Utils.connectAtom(authStore, 'authState')
-)(class SupportRequestModal extends Component {
-  constructor(props) {
-    super(props)
-    const { contactEmail, email } = props.authState.profile
+const SupportRequestModal =
+  Utils.connectAtom(authStore, 'authState')(class SupportRequestModal extends Component {
```

Comment:
Could you move this up to be on the previous line? We try to keep at least the start of the declaration on the same line as the name.